src/pages/en/guides/migrate-to-astro/from-sveltekit.mdx

```diff
@@ -26,11 +26,15 @@ SvelteKit and Astro share some similarities that will help you migrate your proj
 
 When you rebuild your SvelteKit site in Astro, you will notice some important differences:
 
-- [MPA vs SPA](/en/concepts/mpa-vs-spa/): SvelteKit is a Svelte-based SPA (single-page application). Astro sites are multi-page apps built using [`.astro` components](/en/core-concepts/astro-components/), but can also support [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/en/core-concepts/framework-components/) and raw HTML templating.
+- [MPA vs alternatives](/en/concepts/mpa-vs-spa/): Astro sites are multi-page apps, whereas SvelteKit defaults to SPAs (single-page applications) with server-side rendering, but can also create MPAs or traditional SPAs. SvelteKit supports different modes for different pages.
```

Comment:
'Modes' here is basically a shorthand for different combinations of the `ssr` and `csr` [page options](https://kit.svelte.dev/docs/page-options) and whether or not the client-side router is used:

* `ssr = true, csr = true` (default) — server-rendered HTML that is then hydrated
* `ssr = false, csr = true` — app shell with client rendering (traditional SPA)
* `ssr = true, csr = false` — static HTML with no hydration (usually unused, but good for testing your site works if JS fails)
* `ssr = false, csr = false` — nothing is rendered 🙈 

All these options (along with `prerender` and `trailingSlash`) can be set at a page level or at a layout level (and if that's the _root_ layout, they're effectively global, but overrideable, options), so you can mix and match different modes according to need. You can also disable client-side routing with the `data-sveltekit-reload` attribute on any part of the page that contains `<a>` elements.

We think this flexibility is an important differentiator! Perhaps this would be a clearer description?

```suggestion
- [MPA vs alternatives](/en/concepts/mpa-vs-spa/): Astro sites are multi-page apps, whereas SvelteKit defaults to SPAs (single-page applications) with server-side rendering, but can also create MPAs or traditional SPAs; you can mix and match these techniques within an app.
```

Reply:
Thanks for the explanation! More frameworks should come with a see-no-evil mode 😄

I like this change. Suggesting a tiny tweak to avoid the semicolon but otherwise looks great!

```suggestion
- [MPA vs alternatives](/en/concepts/mpa-vs-spa/): Astro sites are multi-page apps, whereas SvelteKit defaults to SPAs (single-page applications) with server-side rendering, but can also create MPAs, traditional SPAs, or you can mix and match these techniques within an app.
```

Reply:
much nicer!